### PR TITLE
Create minisign.rb

### PR DESCRIPTION
**Pull request by reelsense**
A dead simple tool to sign files and verify signatures. Signature written by minisign can be verified using OpenBSD's signify tool: public key files and signature files are compatible."

### CODE REVIEW

File: Library/Formula/minisign.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Minisign < Formula
+  desc "A simple tool to sign files and verify signatures. Signatures are compatible with OpenBSD’s signify tool."
```
Comment (DomT4) [inline Library/Formula/minisign.rb]: The limit is 80 characters here.

File: Library/Formula/minisign.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://jedisct1.github.io/minisign/"
+  url "https://github.com/jedisct1/minisign/archive/0.4.tar.gz"
+  version "0.4"
```
Comment (DomT4) [inline Library/Formula/minisign.rb]: Will be redundant - `brew` can scan fairly simple URLs for the version.

File: Library/Formula/minisign.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "dc7695513e715654a51d07ad3e6b0083f9cb38b1a5bc9f16e1177d15af992dcc"
+
+  # depends_on "cmake" => :build
```
Comment (DomT4) [inline Library/Formula/minisign.rb]: Remove the commented out dupe argument.

File: Library/Formula/minisign.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    #system "false"
```
Comment (DomT4) [inline Library/Formula/minisign.rb]: Please add a test of some kind. If necessary you can write configuration files - The `testpath` is sandboxed and the `HOME` environmental variable pointed to the `testpath` for the duration of said test.

**Comment by reelsense** [comment]
78 characters in description. I'm not sure why it says I'm over 80C.

**Comment by DomT4** [comment]
The `desc` length includes the formula name.

**Comment by reelsense** [comment]
HURRAY!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/37ced5e22fdf9c5dd0aa0af97f40e109bfe5ba35. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Just a nit, but please squash your commits prior to the last commit, otherwise tired maintainers can end up merging 3 commits into the master branch instead of the intended one :stuck_out_tongue_winking_eye:. Thanks!

**Comment by jedisct1** [comment]
Thanks for this, @reelsense !

**Pull request closed without merging**